### PR TITLE
More fixes

### PR DESCRIPTION
**Pull request by devagrawal09**
- [x] Script stopes running if file is not found
- [x] Log files are timestamped to preserve history

### CODE REVIEW

**Review by JacobMGEvans — APPROVED** [review]

File: index.ts [hunk 1]
```diff
@@ -97,20 +99,22 @@ async function processUserToClerk(userData: User) {
...
   await new Promise((r) => setTimeout(r, DELAY));
 }
 
+let spinner: Ora;
+
 async function main() {
   const parsedUserData = JSON.parse(fs.readFileSync("users.json", "utf-8"));
 
+  console.log(`Clerk User Migration Utility`);
+  spinner = ora(`Migrating users...`).start();
```
Comment (JacobMGEvans) [inline index.ts:116]: Why did you move these into the `main` function scope?

**Review by JacobMGEvans — CHANGES_REQUESTED** [review]
Question pending.

File: index.ts (hunk 1 above)
Comment (devagrawal09) [reply index.ts:116]: because the spinner needs to start running after the file check is complete, if not the script will continue running even if the file does not exist and it throws an error that should terminate

Comment (royanger) [reply index.ts:116]: You tested an import after this change? Its fails with no file present, and the import otherwise works?

Comment (devagrawal09) [reply index.ts:116]: Yes

**Review by JacobMGEvans — APPROVED** [review]
